front/components/assistant/HelpDrawer.tsx

```diff
@@ -0,0 +1,211 @@
+import {
+  ArrowRightIcon,
+  Button,
+  ChatBubbleBottomCenterTextIcon,
+  FolderIcon,
+  Item,
+  LightbulbIcon,
+  Modal,
+  Page,
+} from "@dust-tt/sparkle";
+import type {
+  AgentMention,
+  MentionType,
+  UserType,
+  WorkspaceType,
+} from "@dust-tt/types";
+import { isBuilder } from "@dust-tt/types";
+import { useRouter } from "next/router";
+import type { ComponentType } from "react";
+import { useCallback, useContext } from "react";
+
+import { AssistantInputBar } from "@app/components/assistant/conversation/input_bar/InputBar";
+import { createConversationWithMessage } from "@app/components/assistant/conversation/lib";
+import { SendNotificationsContext } from "@app/components/sparkle/Notification";
+import { GLOBAL_AGENTS_SID } from "@app/lib/assistant";
+import { useSubmitFunction } from "@app/lib/client/utils";
+
+const topPicks = [
+  {
+    title: "Creating Assistants",
+    href: "https://www.notion.so/dust-tt/Prompting-101-How-to-Talk-to-Your-Assistants-c0212513056a4af885e41706ee742dda",
+    icon: ArrowRightIcon,
+  },
+  {
+    title: "Managing Connections",
+    href: "https://www.notion.so/dust-tt/Dust-Library-1aa97619d56d4d9f9c21d7ce89d23dbb?pvs=4#4f7f969e52464e369f4cd453457d8059",
+    icon: ArrowRightIcon,
+  },
+  {
+    title: "Choosing a Model",
+    href: "https://blog.dust.tt/comparing-ai-models-claude-gpt4-gemini-mistral/",
+    icon: ArrowRightIcon,
+  },
+];
+
+function LinksList({
+  linksList,
+  title,
+}: {
+  linksList: {
+    title: string;
+    href: string;
+    icon?: ComponentType;
+    description?: string;
+  }[];
+  title?: string;
+}) {
+  return (
+    <Item.List>
+      {title && <Item.SectionHeader label={title} />}
+      {linksList.map((link, index) => (
+        <Item.Navigation
+          icon={link.icon}
+          label={link.title}
+          href={link.href}
+          key={index}
+          description={link.description}
+          hasAction={false}
+        />
+      ))}
+    </Item.List>
+  );
+}
+
+export function HelpDrawer({
+  owner,
+  user,
+  show,
+  onClose,
+}: {
+  owner: WorkspaceType;
+  user: UserType;
+  show: boolean;
+  onClose: () => void;
+}) {
+  const router = useRouter();
+  const sendNotification = useContext(SendNotificationsContext);
+
+  const { submit: handleHelpSubmit } = useSubmitFunction(
+    useCallback(
+      async (input: string, mentions: MentionType[]) => {
+        const inputWithHelp = input.includes("@help")
+          ? input
+          : `@help ${input.trimStart()}`;
+        const mentionsWithHelp = mentions.some(
```

Comment:
Why do we implement all this logic whereas we could either put `@help` in a sticky mention + only suggest this one in the input bar?

Reply:
I tried to use sticky mentions, with the constraint, as discussed with ed, that we want there to be no mention in the input initially and that submitting a message without the mention should work (this is the help, user might not be familiar with mentions)

I could not make it work

Reply:
I'm not entirely convinced about this but as I wasn’t part of your discussion with @Duncid. IMHO, if we’ve agreed to proceed with mentions, we should avoid making exceptions. Exceptions can make it difficult for users to understand how to use our product. In this case, I believe that using a sticky mention for `@help` would not only serve the intended purpose but also help to educate our users about using mentions!

Reply:
Discussed IRL, keeping as is